.github/workflows/maven.yml

```diff
@@ -25,17 +31,25 @@ jobs:
            -Dtest=!com.yahoo.oak.ComputeTest,!com.yahoo.oak.PutIfAbsentTest
     - name: Setup diff-test-coverage
       run: npm install @connectis/diff-test-coverage@1.5.3
-    - name: Validate test-coverage
+    - name: Validate test-coverage compared with master
       run: git diff origin/master HEAD | node_modules/.bin/diff-test-coverage
             --coverage "**/target/site/jacoco/jacoco.xml"
             --type jacoco
             --color
-            --line-coverage 50
```

Comment:
When lines/branch/function coverage specification is removed are there any default values for that requested?

Reply:
I think it is 80%, but it doesn't matter since it doesn't fail anything.